### ISSUE TITLE
Allow to change phpstan rule level

### ISSUE THREAD

**Issue by trahibidadido** [tsufeki/tenkawa-php-language-server#28]
Is it possible to analyse codebase with stan using `--level` argument? In many other editor extensions there is such option, but this awesome ls lacks it.

**Comment by tsufeki** [comment]
I've personally never feel the need for it (IDE is not like CI as there's no binary result, and marking any suspicious may be useful) but I see how this can be distracting - I'll put it in the next iteration.

**Comment by crissi** [comment]
what about config files for phpstan, it does not seem to pick up my config: phpstan.neon, but I might be  mistaking

**Comment by tsufeki** [comment]
Indeed, `phpstan.neon` is ignored, and for good reasons:

1. Versions of phpstan used in your project and bundled in LS may be different, making config file wrong/useless.
2. I can't include extensions "as is" - many of them rely on global state or actually load analysed code, and are therefore impossible to use in a long running process.

Phpstan levels are pretty stable in general so it may be useful to pick them from `phpstan.neon`, but nothing more is viable.

**Comment by crissi** [comment]
That is unfortunate, since I am using Laravel. I also use the config from https://github.com/nunomaduro/ with phpstan vs code plugin to better match Laravel logic, but since your plugin support phpstan + other awesomeness, it would make sense me to only use that instead of phpstan vs code plugin. But right now I get way too many errors with your plugin because the neon file is not supported.

**Comment by trahibidadido** [comment]
@tsufeki `ignoreErrors` and `excludes_analyse` are pretty viable options, in my opinion.

**Comment by yazeed** [comment]
I agree with @crissi, your plugin would be awesome if it supported Larastan (https://github.com/nunomaduro/larastan). Maybe it's a good idea to open a separate issue for it?
